aerospike-proximus/README.md

```diff
@@ -0,0 +1,177 @@
+# Aerospike Proximus
+
+This Helm chart allows you to configure and run our official [Aerospike Proximus](https://hub.docker.com/repository/docker/aerospike/aerospike-proximus)
+docker image on a Kubernetes cluster.
+
+This helm chart sets up a `StatefulSet` for each proximus instance. We use a `StatefulSet` instead of a `Deployment`, to have stable DNS names for the  
+deployed proximus pods.
+
+## Prerequisites
+- Kubernetes cluster
+- Helm v3
+- An Aerospike cluster that can connect to Pods in the Kubernetes cluster.
+  The Aerospike cluster can be deployed in the same Kubernetes cluster using [Aerospike Kubernetes Operator](https://docs.aerospike.com/cloud/kubernetes/operator)
+- Ability to deploy a LoadBalancer on K8s in case proximus app runs outside the Kubernetes cluster
+
+## Adding the helm chart repository
+
+Add the `aerospike` helm repository if not already done
+
+```shell
+helm repo add aerospike https://aerospike.github.io/helm-charts
```

Comment:
I get a 404 on this?

Reply:
to expand on this ```helm search repo aerospike
NAME                               	CHART VERSION	APP VERSION	DESCRIPTION                                       
aerospike/aerospike-graph          	2.0.0        	2.0.0      	Aerospike Graph Service                           
aerospike/aerospike-jms-inbound    	1.0.0        	3.0.0      	A Helm chart for Aerospike Connect for JMS - In...
aerospike/aerospike-jms-outbound   	1.0.0        	4.1.1      	A Helm chart for Aerospike Connect for JMS - Ou...
aerospike/aerospike-kafka-outbound 	1.0.1        	5.1.2      	A Helm chart for Aerospike Connect for Kafka - ...
aerospike/aerospike-pulsar-outbound	1.0.0        	3.2.1      	A Helm chart for Aerospike Connect for Pulsar -...
``` I don't have a proximus to install. We might want to have people run `helm repo update` after installing the new repo as well. though this did not fix my issue.

Reply:
aaaaaannnddd of course it does not exist because this has not been merged yet. dont worry. i'm getting there slowly!